rust/cli/src/main.rs

```diff
@@ -27,7 +27,7 @@ struct Cli {
 fn run(args: RunArgs) {
     let config = match &args.config {
         Some(path) => FrontendConfig::load_from_path(path),
-        None => FrontendConfig::single_node_default(),
+        None => FrontendConfig::single_node_default(None),
```

Comment:
I don't think the change to make the default take a path makes sense. Can you explain why we are doing this?